data/assets/scene/digitaluniverse/starlabels.asset

```diff
@@ -2,7 +2,7 @@ local speck = asset.resource({
   Name = "Star Labels Speck Files",
   Type = "HttpSynchronization",
   Identifier = "digitaluniverse_starlabels_speck",
-  Version = 2
+  Version = 3
```

Comment:
```suggestion
  Version = 4
```
There was already a 3 existing on our data server for some reason

Reply:
ok, noted.